src/vmware/azext_vmware/vendored_sdks/__init__.py

```diff
@@ -1,12 +0,0 @@
-# coding=utf-8
-# --------------------------------------------------------------------------
-# Copyright (c) Microsoft Corporation. All rights reserved.
-# Licensed under the MIT License. See License.txt in the project root for
-# license information.
-#
-# Code generated by Microsoft (R) AutoRest Code Generator.
-# Changes may cause incorrect behavior and will be lost if the code is
-# regenerated.
-# --------------------------------------------------------------------------
-
-__path__ = __import__('pkgutil').extend_path(__path__, __name__)
```

Comment:
Please do not remove this file, it will cause a CI problem in main CLI pipeline [pipeline link](https://dev.azure.com/azclitools/public/_build/results?buildId=19923&view=logs&j=7621d84f-e7df-5fb0-9a4f-5fbdcb3421ee&t=bbcd1db7-629e-5645-8ed8-a18b90e0a11a)
>   File "/opt/az/azcliextensions/vmware/azext_vmware/custom.py", line 8, in <module>
    from azext_vmware.vendored_sdks.avs_client import AVSClient
ModuleNotFoundError: No module named 'azext_vmware.vendored_sdks'

Reply:
It was restored in https://github.com/Azure/azure-cli-extensions/pull/5619